dojo/tools/rusty_hog/parser.py

```diff
@@ -0,0 +1,125 @@
+        items = {}
+        findings = self.__getitem(vulnerabilities=json.load(json_output), scanner=scanner)
+        for finding in findings:
+            unique_key = "Finding {}".format(finding)
+            items[unique_key] = finding
+        return list(items.values())
+
+    def get_tests(self, scan_type, handle):
+        tree = json.load(handle)
+        tests = list()
+        parsername = "Rusty Hog"
+        for node in tree:
+            if 'commit' in node or 'commitHash' in node or 'parent_commit_hash' in node or 'old_file_id' in node or 'new_file_id' in node:
+                parsername = "Choctaw Hog"
+                break
+            if 'issue_id' in node or 'location' in node or 'url' in node:
+                parsername = "Gottingen Hog"
+                break
+        test = ParserTest(
+            name=parsername,
+            type=parsername,
+            version="",
+        )
+        if parsername == "Rusty Hog":  # The outputfile is empty. A subscanner can't be classified
+            test.description = "The exact scanner within Rusty Hog could not be determined due to missing information within the scan result."
+        else:
+            test.description = parsername
+        test.findings = self.__getitem(vulnerabilities=tree, scanner=parsername)
+        tests.append(test)
+        return tests
+
+    def __getitem(self, vulnerabilities, scanner):
+        findings = []
+        line = ""
+        cwe = 200
+        for vulnerability in vulnerabilities:
+            if scanner == "Rusty Hog":
+                break
+            elif scanner == "Choctaw Hog":
+                """Choctaw Hog"""
+                description = "**This string was found:** {}".format(vulnerability.get('stringsFound'))
+                if vulnerability.get('commit') is not None:
+                    description += "\n**Commit message:** {}".format(vulnerability.get('commit'))
+                if vulnerability.get('commitHash') is not None:
+                    description += "\n**Commit hash:** {}".format(vulnerability.get('commitHash'))
+                if vulnerability.get('parent_commit_hash') is not None:
+                    description += "\n**Parent commit hash:** {}".format(vulnerability.get('parent_commit_hash'))
+                if vulnerability.get('old_file_id') is not None and vulnerability.get('new_file_id') is not None:
+                    description += "\n**Old and new file IDs:** {} - {}".format(
+                                    vulnerability.get('old_file_id'),
+                                    vulnerability.get('new_file_id'))
+                if vulnerability.get('old_line_num') is not None and vulnerability.get('new_line_num') is not None:
+                    description += "\n**Old and new line numbers:** {} - {}".format(
+                                    vulnerability.get('old_line_num'),
+                                    vulnerability.get('new_line_num'))
```

Comment:
It would be great to use the line num info in the finding `line` field. Since this would only be applicable for Choctaw Hogs scans, it can be added to the finding after initialization by checking if `line` variable previously defined above is no longer equal to `''`